util/crc32c_ppc.c

```diff
@@ -0,0 +1,101 @@
+/* Copyright (C) 2017 International Business Machines Corp.
+ * All rights reserved.
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
```

Comment:
I can't answer it for sure either but can provide some additional info.
https://github.com/facebook/rocksdb/blob/master/COPYING

Contains the GPL v2 license.

This code originally came from 
https://github.com/antonblanchard/crc32-vpmsum

Which allows license choice for either GPL or Apache 2.0.  If necessary we could probably update these files to choose Apache 2.0 if that is preferred.